WinUIGallery/Controls/PageHeader.xaml

```diff
@@ -48,7 +48,7 @@
                     Visibility="{x:Bind Item.Docs, Converter={StaticResource collectionConverter}}">
                     <DropDownButton.Content>
                         <StackPanel Orientation="Horizontal" Spacing="8">
-                            <FontIcon FontSize="14" Glyph="&#xE8A5;" />
```

Comment:
<img width="281" alt="image" src="https://user-images.githubusercontent.com/9866362/231405303-44364452-2817-470f-aebf-3395c9eccd08.png">

This is still a visible regression over the current version as the text label is not vertically aligned wrt the icon and glyph.

FYI, it's very common to add a specific margin to center align text within a control. (e.g. see the default button padding that has a different top/bottom margin to center align the label)

New vs. old:
<img width="281" alt="image" src="https://user-images.githubusercontent.com/9866362/231405888-38928eae-40ef-4d96-a1c9-878e25202de9.png">

Reply:
Then I would still prefer `VerticalAlignment="Center"`

Reply:
You're comparing the baseline of the icon content with the baseline of text. They don't need to be the same thing.